enterprise/backend/test/metabase_enterprise/serialization/v2/e2e_test.clj

```diff
@@ -735,3 +735,54 @@
                     (is (thrown-with-msg? Exception #"Please upgrade"
                                           (cmd/v2-load! dump-dir {}))
                         "throws")))))))))))
+
+(deftest pivot-export-test
+  (testing "Pivot table export and load correctly"
+    (let [card1s (atom nil)]
+      (ts/with-random-dump-dir [dump-dir "serdesv2-"]
+        (ts/with-source-and-dest-dbs
+          (ts/with-source-db
+            (mt/dataset test-data
+              (mt/$ids nil
+                (t2.with-temp/with-temp
+                  [Database   {db-id :id}    {:name "Pivot Database"}
+                   Collection {coll-id :id}  {:name "Pivot Collection"}
+                   Card       card           {:name          "Pivot Card"
+                                              :collection_id coll-id
+                                              :dataset_query {:type     :query
+                                                              :database db-id
+                                                              :query    {:source-table $$checkins
+                                                                         :aggregation  [:sum [:field %checkins.id nil]]
+                                                                         :breakout     [[:field %checkins.user_id nil]]}}
+                                              :visualization_settings
+                                              {:pivot_table.column_split
+                                               {:rows    [[:field %users.name {:base-type    :type/Text
+                                                                               :source-field %checkins.user_id}]]
+                                                :columns [[:field %venues.name {:base-type    :type/Text
+                                                                                :source-field %checkins.venue_id}]]
+                                                :values  [[:aggregation 0]]}
+                                               :column_settings
+                                               {(format "[\"ref\",[\"field\",%s,null]]" %users.name)
+                                                {:pivot_table.column_sort_order "descending"}}}}]
+                  (reset! card1s card)
+                  (-> (serdes/with-cache (into [] (extract/extract {})))
+                      (storage/store! dump-dir))))))
+
+          (ts/with-dest-db
+            (mt/dataset test-data
+              (mt/$ids nil
```

Comment:
For example in that case there are two children and the removal of `nil` breaks the test (since `load-metabase!` call is used as a table name, haha). So I don't really want to remove nils in both cases - it's less ambiguous this way.

Reply:
Yes, you're right.